### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by KozielPiotr**

### CODE REVIEW

File: shared_bills/settings.py [hunk 1]
```diff
@@ -131,10 +132,48 @@
...
+            "rest_framework.authentication.SessionAuthentication",
+            "rest_framework.authentication.BasicAuthentication",
+        ]
+    )
+
+
+# Simple JWT
+
+SIMPLE_JWT = {
+    "ACCESS_TOKEN_LIFETIME": timedelta(weeks=1),
```
Comment (cmiacz) [inline shared_bills/settings.py:161]: Access token lifetime should be shorter that refresh token lifetime

File: shared_bills/settings.py [hunk 2]
```diff
@@ -131,10 +132,48 @@
...
+    "REFRESH_TOKEN_LIFETIME": timedelta(days=1),
+    "ROTATE_REFRESH_TOKENS": False,
+    "BLACKLIST_AFTER_ROTATION": False,
+    "ALGORITHM": "HS256",
+    "SIGNING_KEY": os.environ["SECRET_KEY"],
+    "VERIFYING_KEY": None,
+    "AUDIENCE": None,
+    "ISSUER": None,
+    "AUTH_HEADER_TYPES": ("Bearer",),
+    "USER_ID_FIELD": "id",
```
Comment (cmiacz) [inline shared_bills/settings.py]: Use email as user identifier

File: bills/views.py [hunk 3]
```diff
@@ -42,7 +58,15 @@ def get_event(self):
...
-    queryset = Event.objects.all()
+    permission_classes = [IsOwner, IsAuthenticated]
+
+    def get_user(self):
+        """returns current user."""
+
+        return self.request.user
+
+    def get_queryset(self):
+        return Event.objects.filter(user=self.get_user())
```
Comment (cmiacz) [inline bills/views.py]: Filter by user too

File: shared_bills/urls.py [hunk 4]
```diff
@@ -7,7 +7,12 @@
 from rest_framework.routers import APIRootView
 
 
-api_root_views = {"events": "events-list", "user": "user-detail"}
+api_root_views = {
+    "events": "events-list",
+    "user": "user-detail",
+    "get token": "token_obtain_pair",
+    "verify token": "token_verify",
```
Comment (cmiacz) [inline shared_bills/urls.py]: Urls are not visible in api root view. Use `-` instead of `_` as in the view names set in ` accounts/urls.py `

**Review by cmiacz — CHANGES_REQUESTED** [review]

File: shared_bills/settings.py (hunk 1 above)
Comment (KozielPiotr) [reply shared_bills/settings.py:161]: Fixed

File: shared_bills/settings.py (hunk 2 above)
Comment (KozielPiotr) [reply shared_bills/settings.py]: Done

File: shared_bills/urls.py (hunk 4 above)
Comment (KozielPiotr) [reply shared_bills/urls.py]: Fixed

File: shared_bills/settings.py [hunk 5]
```diff
@@ -168,7 +168,7 @@
     "AUDIENCE": None,
     "ISSUER": None,
     "AUTH_HEADER_TYPES": ("Bearer",),
-    "USER_ID_FIELD": "id",
+    "USER_ID_FIELD": "email",
     "USER_ID_CLAIM": "user_id",
```
Comment (cmiacz) [inline shared_bills/settings.py]: Change it for email too

**Review by cmiacz — APPROVED** [review]

**Pull request merged**